packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -118,6 +127,18 @@ export default class Server {
         if (await this.context.isLWCTemplate(doc)) {
             this.auraDataProvider.activated = false; // provide completions for lwc components in an Aura template
             this.lwcDataProvider.activated = true;
+        } else if (await this.context.isLWCJavascript(doc)) {
```

Comment:
If the completion event is triggered from a lwc js file we get the already cached tag info and add their `c/<component name>` info to the rest of the suggestions provided by the html lang server.